frontend/libs/studio-components/src/components/StudioCodeFragment/StudioCodeFragment.module.css

```diff
@@ -1,8 +1,12 @@
 .code {
-  background-color: var(--fds-semantic-surface-neutral-subtle);
+  flex: 1;
+  background-color: rgb(221, 221, 234);
```

Comment:
Maybe we can use a variable from DS here? 🫣

Reply:
Was the variable `--fds-semantic-surface-neutral-subtle` wrong? 😊

Reply:
no not wrong , it was just test for background I forget to set it back🙈

Reply:
Fixed